README.md

```diff
@@ -87,7 +88,7 @@ Runs:
 * __js(gulp, config)__ Config accepts:
 	- js: `String` Path to your main JavaScript file. (Default: './main.js' and checks your bower.json to see if it's in its main key)
 	- buildJs: `String` Name of the built JavaScript bundle. (Default: 'main.js')
-	- buildFolder: `String` Path to directory where the built file will be created. (Default: './build/')
+	- buildFolder: `String` Path to directory where the built file will be created. If set to `null`, files won't be saved. (Default: './build/')
```

Comment:
it's if null. I can't make it undefined because by default I want it to be saved in a default folder, and the default includes `undefined`, that's why I decided on null